images/linux/scripts/installers/haskell.sh

```diff
@@ -53,10 +55,11 @@ if ! command -v stack; then
     exit 1
 fi
 
+# Add the latest ghc to path
+prependEtcEnvironmentPath "/opt/ghc/$defaultGHCVersion/bin"
```

Comment:
May be it is better to make symlink to `/usr/bin/ghc`?

Reply:
I suppose this symlink may cause some issues for `setup-haskell` action task.

Reply:
What issue do you mean?
Adding something to the PATH is always dangerous. `/opt/ghc/$defaultGHCVersion/bin` contains a lot of binaries:
![image](https://user-images.githubusercontent.com/16715858/88765338-026d7100-d17f-11ea-8f73-bd361cf73ccc.png)
And all of them will be added to the PATH that could cause customers impact.

Reply:
Yes, but `setup-haskell` task add `/opt/ghc/<version>/bin` directory to path for installed ghc and it works correctly. I'm afraid using symlink in `/usr/bin` may cause some conflict with ghc version that will be installed by `setup-haskell`

Reply:
`setup-haskell` does it only for customers who need ghc in path so it is expected.
Symlink should not cause issues because `setup-haskell` will continue to add directory to PATH and this directory will be before `/usr/bin` in PATH

Reply:
In other words, I decided to act similarly to the `setup-haskell` task.

Reply:
Done, what about other haskell tool `cabal`? We need to create symlink for it?

Reply:
Yes, I think we should. Good point